src/main/java/org/stellar/sdk/Sep10Challenge.java

```diff
@@ -157,11 +162,6 @@ public static ChallengeTransaction readChallengeTransaction(String challengeXdr,
       throw new InvalidSep10ChallengeException("The transaction sequence number should be zero.");
     }
 
-    // verify that transaction has time bounds set, and that current time is between the minimum and maximum bounds.
```

Comment:
Why gone?

Reply:
looked redundant, the test will fail on L165 if it were null